pkg/web/src/app/common/components/SelectOpenShiftNetworkModal.tsx

```diff
@@ -0,0 +1,158 @@
+import * as React from 'react';
```

Comment:
Same here, this file is being considered deleted/rewritten, if we can `git mv` instead to retain history that would be great

Reply:
same